docs/tables/ibm_is_flow_log.md

```diff
@@ -0,0 +1,63 @@
+# Table: ibm_is_flow_log
+
+Flow Logs for VPC enable the collection, storage, and presentation of information about the Internet Protocol (IP) traffic going to and from network interfaces within your Virtual Private Cloud (VPC).
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  name,
+  id,
+  crn,
+  lifecycle_state,
+  created_at
+from
+  ibm_is_flow_log;
+```
+
+### List flow log collectors by name
+
+```sql
+select
+  name,
+  id,
+  crn,
+  lifecycle_state,
+  created_at
+from
+  ibm_is_flow_log
+where
+  name = 'steampipe01';
+```
+
+### List all inactive flow log collectors
+
+```sql
+select
+  name,
+  id,
+  crn,
+  lifecycle_state,
+  created_at
+from
+  ibm_is_flow_log
+where
+  not active;
+```
+
+### List all flow log collectors with auto delete disabled
```

Comment:
Can you pls match it with [AWS](https://hub.steampipe.io/plugins/turbot/aws/tables/aws_vpc_flow_log)